packages/dropdowns/demo/select.stories.mdx

```diff
@@ -22,6 +22,10 @@ import README from '../README.md';
 
 # API
 
+**DEPRECATED:** use `@zendeskgarden/react-dropdowns.next`
+[Combobox](https://zendeskgarden.github.io/react-components/?path=/docs/packages-dropdowns-next-combobox)
+instead.
```

Comment:
Should we be linking to the `Select` component (in `forms`) instead?

Reply:
Nope. `Combobox` is the [replacement](https://garden.zendesk.com/components/combobox#select-only) for this custom `Select`. The `Select` in forms is a native `<select>` with no listbox styling.